### PR TITLE
tiflash: Add label schedule for TiFlash

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7064, https://github.com/pingcap/docs-cn/pull/8863
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by LittleFall — APPROVED** [review]

File: tiflash/use-tiflash.md [hunk 1]
```diff
@@ -138,6 +142,74 @@ To check tables without TiFlash replicas in the database, you can execute the fo
 SELECT TABLE_NAME FROM information_schema.tables where TABLE_SCHEMA = "<db_name>" and TABLE_NAME not in (SELECT TABLE_NAME FROM information_schema.tiflash_replica where TABLE_SCHEMA = "<db_name>")
 ```
 
+### Set available zones
+
+When configuring replicas, if you need to distribute TiFlash replicas to multiple data centers for the purpose of disaster recovery, you can configure available zones by following steps below:
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:147]:
```suggestion
When configuring replicas, if you need to distribute TiFlash replicas to multiple data centers for disaster recovery, you can configure available zones by following the steps below:
```

File: tiflash/use-tiflash.md [hunk 2]
```diff
@@ -138,6 +142,74 @@ To check tables without TiFlash replicas in the database, you can execute the fo
...
+        config:
+          flash.proxy.labels: zone=z1
+      - host: 172.16.5.85
+        config:
+          flash.proxy.labels: zone=z2
+    ```
+
+2. After starting a cluster, specify the labels when creating replicas.
+
+     {{< copyable "sql" >}}
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:166]:
```suggestion
    {{< copyable "sql" >}}
```

File: tiflash/use-tiflash.md [hunk 3]
```diff
@@ -138,6 +142,74 @@ To check tables without TiFlash replicas in the database, you can execute the fo
...
+
+    ```sql
+    ALTER TABLE table_name SET TIFLASH REPLICA count LOCATION LABELS location_labels
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:169]:
```suggestion
    ALTER TABLE table_name SET TIFLASH REPLICA count LOCATION LABELS location_labels;
```

File: tiflash/use-tiflash.md [hunk 4]
```diff
@@ -138,6 +142,74 @@ To check tables without TiFlash replicas in the database, you can execute the fo
...
+
+    For example:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    ALTER TABLE t SET TIFLASH REPLICA 2 LOCATION LABELS "zone";
+    ```
+
+3. The PD schedules the replicas based on the labels. In this example, the PD schedules two replicas of table `t` to two available zones, respectively. You can use pd-ctl to view the scheduling.
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:180]:
```suggestion
3. PD schedules the replicas based on the labels. In this example, PD respectively schedules two replicas of the table `t` to two available zones. You can use pd-ctl to view the scheduling.
```

File: tiflash/use-tiflash.md [hunk 5]
```diff
@@ -14,7 +14,11 @@ You can either use TiDB to read TiFlash replicas for medium-scale analytical pro
 - [Use TiDB to read TiFlash replicas](#use-tidb-to-read-tiflash-replicas)
 - [Use TiSpark to read TiFlash replicas](#use-tispark-to-read-tiflash-replicas)
 
-## Create TiFlash replicas for tables
+## Create TiFlash replicas
+
+This section describes how to create TiFlash replicas for tables and for databases, and set available zones.
```
Comment (TomShawn) [inline tiflash/use-tiflash.md:19]:
```suggestion
This section describes how to create TiFlash replicas for tables and for databases, and set available zones for replica scheduling.
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

**Pull request merged**